### DISCUSSION TITLE
예약신청페이지

### DISCUSSION THREAD

**Discussion by Kimkim19** [kimytsc/covid-rest-vaccine-macro#56] [Q&A]
제가 이해를 못하겠어서요ㅠㅠ 저는 모바일에서 하고있어요!! 알려주신 순서대로 하고 있는데 
1. 예약신청페이지를 켜놓고 소스는 크롬에서 붙여넣기 하는거 맞나요?? 
2. 예약신청페이지에서 분당성모병원?신창 페이지가 나오는게 맞을까요?
3. 크롬어플에서 javascript: 치고 복사넣기하면 에러페이지가 뜹니다ㅠㅠ
![700CB214-6B72-4BBD-AE69-6EC49865BDE0](https://user-images.githubusercontent.com/89031295/129590671-0e42709f-6254-41ef-812f-94f2ad283bd6.png)
![84019F8E-ACD7-41C0-94E4-3D27EFD6787E](https://user-images.githubusercontent.com/89031295/129590679-044b905c-04e5-426e-a42c-21cba5b5e328.png) 

혹시몰라 코드 같이 첨부합니다!!
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=unused&x=127.1666716&y=37.5599045&bounds=127.158625%3B37.5500552%3B127.1747182%3B37.569752",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",/*화이자*/
        "VEN00014",/*모더나*/
      /*"VEN00015",/*아스트라제네카, 2021-08-09 기준 50세 이상만 신청 가능하여 주석처리, 필요시 '/*"VEN00015'의 '/*' 제거*/
      /*"VEN00016",/*얀센, 기본값에서 제거, 필요시 '/*"VEN00016'의 '/*' 제거*/
      /*"VEN00017",/*노바백스, 기본값에서 제거, 필요시 '/*"VEN00017'의 '/*' 제거*/
      /*"VEN00018",/*시노팜, 기본값에서 제거, 필요시 '/*"VEN00018'의 '/*' 제거*/
      /*"VEN00019",/*시노백, 기본값에서 제거, 필요시 '/*"VEN00019'의 '/*' 제거*/
      /*"VEN00020",/*스푸트니크V, 기본값에서 제거, 필요시 '/*"VEN00020'의 '/*' 제거*/
    ]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/kakao/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()

**Comment by kimytsc** [comment]
> 1. 예약신청페이지를 켜놓고 소스는 크롬에서 붙여넣기 하는거 맞나요??

네, 예약신청 페이지에서 복사한 코드(=첨부하신 코드)를 `javascript:`를 입력 후 붙여넣기 하는게 맞습니다.

> 2. 예약신청페이지에서 분당성모병원?신창 페이지가 나오는게 맞을까요?

네, README의 예시로 적어놓은 `예약신청 페이지`가 `분당성모이비인후과의원`이며, 해당 병원의 예약신청 페이지가 노출된 것은 정상입니다.

> 3. 크롬어플에서 javascript: 치고 복사넣기하면 에러페이지가 뜹니다ㅠㅠ

우선. 복사하신 코드는 카카오의 예약시도 코드이므로 카카오 로그인 이후에 사용 가능하며, 네이버의 코드를 복사하신 후에 다시 시도해보시길 바랍니다.
단, 카카오 잔여백신 예약 시도 하실때는 질문에 붙여넣으신 코드를 다시 복사하지 말고, README의 코드를 복사 후 사용해 주시길 바랍니다. 질문의 코드를 복사하면, 특정 문자(`*`)가 없는 코드로 오류가 발생하기 때문입니다.
